ts/Series/Spline/SplineSeries.ts

```diff
@@ -62,7 +62,7 @@ class SplineSeries extends LineSeries {
      * @product      highcharts highstock
      * @optionparent plotOptions.spline
      */
-    public static defaultOptions: SplineSeriesOptions = merge(LineSeries.defaultOptions);
+    public static defaultOptions: SplineSeriesOptions = merge(LineSeries.defaultOptions, {});
```

Comment:
As above

Reply:
Here it is because of the JSDoc parser, as it needs curly brackets.

Reply:
What's stopping future us from removing the empty object because we know it doesn't make a difference logically, then breaking JSDoc? Can we fix JSDoc instead? Are there more cases like this?

Reply:
That is the only one, I am aware of. We might get JSDoc fixed instead. I will give it a try tomorrow.